src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_validators.py

```diff
@@ -197,9 +197,16 @@ def base64_encoded_certificate_type(string):
     return cert_data
 
 def datetime_type(string):
-    """ Validates UTC datettime in format '%Y-%m-%d\'T\'%H:%M\'Z\''. """
-    date_format = '%Y-%m-%dT%H:%MZ'
-    return datetime.strptime(string, date_format)
+    """ Validates UTC datettime in accepted format. Examples: 2017-12-31T01:11:59Z,
+    2017-12-31T01:11Z or 2017-12-31T01Z or 2017-12-31 """
```

Comment:
All looks fine exception the format: "2017-12-31T01Z" looks a bit unconventional. It is not one of the format accpetable to Azure Restful API

 @tjprescott thoughts?